testing/README.md

```diff
@@ -11,43 +11,32 @@ contains a `name` property and a `fn` property. When running tests and
 outputting the results, the name of the past function is used, or if the
 object is passed, the `name` property is used to identify the test.
 
-The module also exports `assert`, `assertEqual`, and `equal`.
-
-`equal` is a deep comparision function, where `actual` and `expected` are
-compared deeply, and if they vary, `equal` returns `false`.
-
-The export `assert` is a function, but it is also decorated with other useful
-functions:
+Asserts are exposed in `testing/asserts.ts` module.
 
+- `equal` - Deep comparision function, where `actual` and `expected` are
+  compared deeply, and if they vary, `equal` returns `false`.
 - `assert()` - Expects a boolean value, throws if the value is `false`.
-- `assert.equal()` - Uses the `equal` comparison and throws if the `actual` and
+- `assertEq()` - Uses the `equal` comparison and throws if the `actual` and
   `expected` are not equal.
-- `assert.strictEqual()` - Compares `actual` and `expected` strictly, therefore
+- `assertStrictEq()` - Compares `actual` and `expected` strictly, therefore
   for non-primitives the values must reference the same instance.
-- `assert.throws()` - Expects the passed `fn` to throw. If `fn` does not throw,
+- `assertThrows()` - Expects the passed `fn` to throw. If `fn` does not throw,
   this function does. Also compares any errors thrown to an optional expected
   `Error` class and checks that the error `.message` includes an optional
   string.
-- `assert.throwsAsync()` - Expects the passed `fn` to be async and throw (or
+- `assertThrowsAsync()` - Expects the passed `fn` to be async and throw (or
   return a `Promise` that rejects). If the `fn` does not throw or reject, this
   function will throw asynchronously. Also compares any errors thrown to an
   optional expected `Error` class and checks that the error `.message` includes
   an optional string.
 
-`assertEqual()` is the same as `assert.equal()` but maintained for backwards
-compatibility.
-
 `runTests()` executes the declared tests.
 
 Basic usage:
 
 ```ts
-import {
-  runTests,
-  test,
-  assert,
-  equal
-} from "https://deno.land/x/testing/mod.ts";
+import { runTests, test, equal } from "https://deno.land/x/testing/mod.ts";
+import { assert } from "https://deno.land/x/testing/asserts.ts";
```

Comment:
1. Use https://deno.land/std/testing/asserts.ts  (without /x/) 

2. Use `assertEq("hello", "world")` here rather than `assert(equal("hello", "world"))`